connectors/aliyun/rocketmq-connect-eventbridge/src/main/java/org/apache/rocketmq/connect/eventbridge/sink/EventBridgeSinkConnector.java

```diff
@@ -0,0 +1,100 @@
+package org.apache.rocketmq.connect.eventbridge.sink;
+
+import io.openmessaging.KeyValue;
+import io.openmessaging.connector.api.component.task.Task;
+import io.openmessaging.connector.api.component.task.sink.SinkConnector;
+import io.openmessaging.internal.DefaultKeyValue;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.rocketmq.connect.eventbridge.sink.constant.EventBridgeConstant;
+import org.apache.rocketmq.connect.eventbridge.sink.utils.DateUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class EventBridgeSinkConnector extends SinkConnector {
+
+    private String accessKeyId;
+
+    private String accessKeySecret;
+
+    private String roleArn;
+
+    private String roleSessionName;
+
+    private String eventTime;
+
+    private String eventSubject;
+
+    private String eventType;
+
+    private String aliyuneventbusname;
+
+    private String accountEndpoint;
+
+    private String stsEndpoint;
+
+    @Override
+    public void pause() {
+
+    }
+
+    @Override
+    public void resume() {
+
+    }
+
+    @Override
+    public List<KeyValue> taskConfigs(int maxTasks) {
+        List<KeyValue> keyValueList = new ArrayList<>(11);
+        KeyValue keyValue = new DefaultKeyValue();
+        keyValue.put(EventBridgeConstant.ACCESS_KEY_ID, accessKeyId);
+        keyValue.put(EventBridgeConstant.ACCESS_KEY_SECRET, accessKeySecret);
+        keyValue.put(EventBridgeConstant.ROLE_ARN, roleArn);
+        keyValue.put(EventBridgeConstant.ROLE_SESSION_NAME, roleSessionName);
+        keyValue.put(EventBridgeConstant.EVENT_TIME, eventTime);
+        keyValue.put(EventBridgeConstant.EVENT_SUBJECT, eventSubject);
+        keyValue.put(EventBridgeConstant.ALIYUN_EVENT_BUS_NAME, aliyuneventbusname);
+        keyValue.put(EventBridgeConstant.ACCOUNT_ENDPOINT, accountEndpoint);
+        keyValue.put(EventBridgeConstant.STS_ENDPOINT, stsEndpoint);
+        keyValue.put(EventBridgeConstant.EVENT_TYPE, eventType);
+        keyValueList.add(keyValue);
+        return keyValueList;
+    }
+
+    @Override
+    public Class<? extends Task> taskClass() {
+        return EventBridgeSinkTask.class;
+    }
+
+    @Override
+    public void validate(KeyValue config) {
+        if (StringUtils.isBlank(config.getString(EventBridgeConstant.ACCESS_KEY_ID))
+                || StringUtils.isBlank(config.getString(EventBridgeConstant.ACCESS_KEY_SECRET))
+                || StringUtils.isBlank(config.getString(EventBridgeConstant.ACCOUNT_ENDPOINT))
+                || StringUtils.isBlank(config.getString(EventBridgeConstant.ROLE_ARN))
```

Comment:
ROLE ARN \ROLE SESSION NAME\ STS is not necessary  when in ak mode directly.

Reply:
processed